packages/desktop-client/src/components/modals/ImportTransactions.js

```diff
@@ -185,29 +185,29 @@ function getInitialMappings(transactions) {
   }
 
   let dateField = key(
-    fields.find(([name, value]) => name.toLowerCase().includes('date')) ||
-      fields.find(([name, value]) => value.match(/^\d+[-/]\d+[-/]\d+$/)),
+    fields.find(([name]) => name.toLowerCase().includes('date')) ||
+      fields.find(([value]) => value.match(/^\d+[-/]\d+[-/]\d+$/)),
```

Comment:
```suggestion
      fields.find(([, value]) => value.match(/^\d+[-/]\d+[-/]\d+$/)),
```

Reply:
Can you help me understand this suggestion?